reana_client/cli/workflow.py

```diff
@@ -198,8 +201,19 @@ def workflow_workflows(ctx, sessions, _filter, output_format, access_token,
          "submitting it's contents to REANA server.")
 @add_access_token_options
 @click.pass_context
-def workflow_create(ctx, file, name, skip_validation, access_token):
-    """Create a REANA compatible workflow from REANA spec file."""
+def workflow_create(ctx, file, name,
+                    skip_validation, access_token):  # noqa: D301
+    """Create a new analysis workflow.
+
+    The `create` command permits to create a new workflow from specifications
+    file. By default, if no file is passed with `-f` or `--file` (see 1st
```

Comment:
Could be shorter:

The `create` command permits to create a new workflow from `reana.yaml` specifications file. The file is expected to be located in the current working directory, or supplied via command-line `-f` option, see examples below.